packages/editor-ui/index.html

```diff
@@ -8,6 +8,9 @@
 		<script type="text/javascript">
 			window.BASE_PATH = '/{{BASE_PATH}}/';
 		</script>
+		<script src="https://public.n8n.cloud/posthog-hooks.js"></script>
```

Comment:
I think we can omit this line since the script below will add it again
<img width="841" alt="CleanShot 2023-01-12 at 16 04 07@2x" src="https://user-images.githubusercontent.com/12657221/212102355-833355b4-6209-469f-a96d-dd6f0ab791ce.png">

Also, we already have CLI function that injects the same snippet and it accepts several params. IMO doing this again on the FE might lead to inconsistencies.
packages/cli/src/commands/start.ts#180 calls `createPostHogLoadingScript`
<img width="1130" alt="CleanShot 2023-01-12 at 16 46 26@2x" src="https://user-images.githubusercontent.com/12657221/212113372-d918702d-9811-4f66-ab4a-1020e4558d09.png">

Reply:
this would only affect dev process 8080.. it does not work for me when running `npm run dev` in editor-ui.. does it work for you? am I missing something?

Reply:
It should only work if you have the diagnostic env variable enabled(`N8N_DIAGNOSTICS_ENABLED`). I've just tested it by commenting out your changes and adding `data-test` attribute to the CLI `createPostHogLoadingScript` method and I'm able to see the right output on both 5678 and 8080 🤔

Reply:
And with your changes, I see this snippet twice:
<img width="929" alt="CleanShot 2023-01-12 at 17 20 40@2x" src="https://user-images.githubusercontent.com/12657221/212122783-33939ad1-d5a3-4b56-807a-e4ef8908ba1e.png">

Reply:
weird was not getting it

Reply:
removed it for now..